app/src/providers/stable-diffusion/client.ts

```diff
@@ -0,0 +1,49 @@
+import axios, { Axios } from "axios";
+import { PromptCreationRequestBody, StableDiffusionConfigOptions } from "./client.types";
+import { Prediction } from "replicate";
+
+export class StableDiffusionClient {
+  private readonly httpClient: Axios | undefined = undefined;
+  private readonly clientConfig: StableDiffusionConfigOptions;
+
+  constructor(clientConfig?: StableDiffusionConfigOptions) {
+    this.clientConfig = clientConfig ?? {
+      baseURL: "https://api.replicate.com/v1",
+      auth: process.env.REPLICATE_API_TOKEN ?? "",
+      modelVersion: process.env.REPLICATE_MODEL ?? "",
+    };
+
+    if (!this.httpClient) {
+      this.httpClient = axios.create({
+        baseURL: clientConfig?.baseURL,
+        headers: {
+          Authorization: `Token ${clientConfig?.auth}`,
+        },
+      });
+    }
+  }
+
+  async create({ prompt }: PromptCreationRequestBody) {
```

Comment:
This args object should also include `negative_prompt`: https://github.com/aufacicenta/pulsemarkets/blob/prompt-wars/app/src/pages/api/prompt-wars/reveal.ts#L88-L96